azext_iot/_help.py

```diff
@@ -912,3 +912,253 @@
     type: command
     short-summary: Delete a device registration in an Azure IoT Hub Device Provisioning Service.
 """
+
+helps['iot dt'] = """
+    type: group
+    short-summary: Manage digital twin of an IoT Plug and Play device.
+"""
+
+helps['iot dt invoke-command'] = """
+    type: command
+    short-summary: Executes a command on an IoT Plug and Play device.
+    long-summary: You can leverage az login and provide --hub-name instead of --login for every command.
+    examples:
+    - name: Execute a command on device .
+      text: >
+        az iot dt invoke-command --login {iothub_cs}
+        --interface {plug_and_play_interface} --device-id {device_id}
+        --command-name {command_name} --command-payload {payload}
+    - name: Execute a command on device within current session.
+      text: >
+        az iot dt invoke-command --hub-name {iothub_name}
+        --interface {plug_and_play_interface} --device-id {device_id}
+        --command-name {command_name} --command-payload {payload}
+"""
+
+helps['iot dt list-interfaces'] = """
```

Comment:
Why is digitaltwin in respective commands shortened to 'dt' but capability-model (like `iot pnp capability-model create`) spelled out? Also in this [PnP CLI article](https://github.com/MicrosoftDocs/azure-docs-pr/pull/81679/files), the digitaltwin example commands are spelled out, like `az iot digitaltwin list-devices`. Let's be consistent with our command set in the extension and also make sure docs are accurate.

Reply:
According to the latest discussion with Ilie, we have to use "dt"(digitaltwin) in commands which are interacting with digitaltwin instances and use "pnp" for model repo commands. As far as concern with "capability-model", this was asked me to use this instead of "model".

> **Previous**: az iot pnp model create

> **Now**: az iot pnp capability-model create

I don't have the access on "PnP CLI article"

Reply:
Thanks for your explanation @anusapan. If that is the decision then the document needs to be changed to reflect this update. Also in general we should be mindful how long our commands get.

Reply:
+@iluican

Reply:
Ok the [latest preview docs](https://github.com/MicrosoftDocs/azure-docs-pr/blob/release-preview-adb-pnp/articles/iot-pnp/howto-install-pnp-cli.md) have 'dt' as you have here, but `capability-model` is still `model`.